src/gramps.js

```diff
@@ -97,13 +97,14 @@ export default function gramps(
 
   const schema = mergeSchemas({ schemas });
 
-  const context = sources.reduce(
-    (models, source) => ({
+  const context = sources.reduce((models, source) => {
```

Comment:
Code Climate is warning us about complexity, and I'm inclined to agree. Probably worth pulling this function out as a helper so it's named and a bit easier to understand, e.g.

```js
const context = sources.reduce(getDataSourceContext);
```